app/controllers/menu_days_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class MenuDaysController < ApplicationController
+  def index
+    @menu_days = MenuDays.find_by_day(1).dishes
```

Comment:
Nope. It should be done in show method, because logic in index looks like list of days.
https://edgeguides.rubyonrails.org/routing.html#singular-resources

Reply:
Also read about magic numbers